tests/test_scripts.py

```diff
@@ -2693,6 +2693,17 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting():
```

Comment:
I thought that was the purpose of this test, to ensure that gzipped
counting tables (still) work, regardless of gzip library or other code
changes.

On Wed Feb 25 2015 at 1:17:22 PM C. Titus Brown <notifications@github.com>
wrote:

> In tests/test_scripts.py
> <https://github.com/ged-lab/khmer/pull/809#discussion_r25363825>:
>
> > @@ -2693,6 +2693,17 @@ def test_trim_low_abund_trimtest_savetable():
> >                  'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
> >
> >
> > +def test_counting():
>
> On Wed, Feb 25, 2015 at 08:38:25AM -0800, Michael R. Crusoe wrote: > @@
> -2693,6 +2693,17 @@ def test_trim_low_abund_trimtest_savetable(): >
> 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA' > > > +def test_counting():
> @ctb <https://github.com/ctb> If the counting file format changes then
> shouldn't this test fail?
> that's a test with a different purpose, I think.
> … <#msg-f:1494103614401295534_>
> --titus
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ged-lab/khmer/pull/809/files#r25363825>.
>